pkg/kubectl/cmd/scale.go

```diff
@@ -120,10 +115,18 @@ func RunScale(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []stri
 		Flatten().
 		Do()
 	err = r.Err()
+	if resource.IsUsageError(err) {
+		return cmdutil.UsageError(cmd, err.Error())
+	}
 	if err != nil {
 		return err
 	}
 
+	count := cmdutil.GetFlagInt(cmd, "replicas")
+	if count < 0 {
+		return cmdutil.UsageError(cmd, "The --replicas=<count> flag is required, and <count> must be greater than or equal to 0")
```

Comment:
/me must add relevant POSIX documentation links to [kubectl-conventions.md](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/kubectl-conventions.md).

https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html
http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html